### PR TITLE
Add my photo

### PR DESCRIPTION
**Pull request by ncdai**

### CODE REVIEW

**Review by abishekvashok — APPROVED** [review]

File: Gemfile.lock [hunk 1]
```diff
@@ -0,0 +1,238 @@
+GEM
```
Comment (poush) [inline Gemfile.lock]: Please remove this file.

**Comment by ParthS007** [comment]
@ncdai3651408 Please resolve conflicts and remove `Gemfile.lock`

**Review by isuruAb — COMMENTED** [review]
hello @ncdai3651408  please resolve conflict

**Comment by ncdai** [comment]
I resolved conflict

**Review by anshumanv — APPROVED** [review]

**Comment by anshumanv** [comment]
Please squash your commits form next time. :smile:

**Comment by ncdai** [comment]
@anshumanv OK, thank you!! 😊

**Review by codedsun — COMMENTED** [review]
Nice work 👍

**Pull request merged**